### PR TITLE
Adding issue templates

### PR DESCRIPTION
**Pull request by RedBrogdon**
One for bugs, and a blank one for everything else.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Version:
+
+### Your code
+
+What code was in the editor, if any, when the failure occurred? You can paste it in below:
+
+```
+// code goes here
+```
```
Comment (johnpryan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should we add a console output section?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+<!-- 
+    Thank you for using DartPad!
+
+    If you've run into a problem while trying to edit or run code in DartPad,
+    we'd like to help. By providing as much information as possible about
+    your browser, machine, and experience, you can help us identify 
+    problems as quickly as possible.
+
+    Please looks through the questions below and answer as many as you can.
```
Comment (johnpryan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We could also add a link to https://dart.dev/tools/dartpad/troubleshoot

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (RedBrogdon) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good idea. Done.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (RedBrogdon) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done.

**Pull request merged**